src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/entity/response/PlanVersionResponse.kt

```diff
@@ -22,5 +22,12 @@ data class PlanVersionResponse(
         planVersion = planVersionEntity.version.toLong(),
       )
     }
+
+    fun from(planVersionEntity: PlanVersionEntity, planUuid: UUID): PlanVersionResponse {
```

Comment:
I don't think we need this. CoordinatorController can just return 
```
      return planService.signPlan(planUuid, signRequest)
        .let { PlanVersionResponse.from(it) }
```

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/117/commits/87a804e371495a39747b67c25c4893ed37298f0d